WordPress/Tests/Theme/NoDeregisterCoreScriptUnitTest.inc

```diff
@@ -22,6 +22,9 @@ wp_deregister_script( 'themename-' . 'common' );
  */
 wp_deregister_script( $prefix . '-common' ); // Warning.
 wp_deregister_script( $something, 'jquery' ); // Warning for variable. Having the handle as second param will not work anyway, so no error.
+wp_deregister_script( self::$script_handle, 'jquery' ); // Warning for variable/dynamic param.
+wp_deregister_script( parent::SCRIPT_HANDLE, 'jquery' ); // Warning for variable/dynamic param.
+wp_deregister_script( get_script_handle(), 'jquery' ); // Warning for variable/dynamic param.
```

Comment:
The second argument is not valid. Is there a reason to add it?

Reply:
Ah, that was only meant to be in the test on line 24, will remove them from the others. And yes, it is just testing that the correct param is being checked.